server.js

```diff
@@ -230,7 +230,7 @@ if (config.devMode) {
 }
 
 // clear cookies on the homepage to reset any stale session state
-app.use(/^\/pl\/?/, require('./middlewares/clearCookies'));
+app.use(/^(?:\/?)$|^(?:\/pl\/?)$/, require('./middlewares/clearCookies'));
```

Comment:
Why are we using non-capturing parentheses (ones that start with `?:`) here?

Reply:
habit - reverted to capturing